2020-06-23-Austin/agenda.md

```diff
@@ -0,0 +1,129 @@
+# OpenJS Collaborator Summit "Austin" 2020 - June 22, 25-26 
+
+This is a live document that will be changed as sessions are added to the agenda.
+
+A collaborator summit is a mix of presentations and interactive sessions lead by the Teams, Working Groups, and Projects that participate in the summit. For interactive sessions, our time is best served if a the team/working groups/project participate in good numbers (quorum etc).
+
+### Registration:
+Attendees must register [here](http://www.cvent.com/d/pnqqmy/4W?ct=50221cf5-5496-4c34-9ec0-3b52b1bf1204&_ga=2.175784992.346531408.1591736714-420567812.1591736714).
+
+### Location:
+This is an online only event - we are using the [QiQoChat Platform](https://collabsummit.qiqochat.com/). If you are a session presenter, please sign up for the platform in advance.
+
+### Timing:
+- Sessions begin no earlier than 9 a.m. GMT-5 (US Central). Timing for each session has been determined based on the availability and timezone of the session host(s).
+
+### Async Chat via OpenJS Foundation Slack:
+Link to join slack: https://communityinviter.com/apps/js-foundation/join-openjs-foundation-on-slack .
+Once you join the OpenJS Foundation Slack Workspace, you can participate in any of the public channels. If you are new to Slack, you can view a full list of available channels by clicking the “Channels” link on the left.
+
+You can ask Collab Summit specific questions of the organizers in #collabsummit.
```

Comment:
I think it would be helpful to create a [Slack User Group](https://slack.com/intl/en-ca/help/articles/212906697-Create-a-user-group) that includes people listed in the following section. This should be opt-in.

That would allow mention of the User Group to notify all the organizers, without participants having to look up individual handles.